traffic_detector.py

```diff
@@ -0,0 +1,35 @@
+import cv2
+import numpy as np
+import time
+
+
+def car_detection():
+    """
+    Method to detect car objects from the traffic video
+    """
+    car_classification = cv2.CascadeClassifier("C:\\Users\\bsens\\Downloads\\haarcascade_car.xml")
```

Comment:
Avoid using hard-coded paths in the code. I can't verify if this code even works because of the hard-coded paths. Consider taking the video file and classifier as command line arguments. Since the code requires a pre-trained haar cascade xml file to work, there should be some instructions on how to get one or a link to download it.

Reply:
Thanks for the review. I will check that. Actually, Darknet is not working at my end. Thus, taking this only.